### PR TITLE
Update CHANGELOG & README for Chat & Common for release

### PR DESCRIPTION
**Pull request by CHILIU-MSFT**
- Updated version number & date in CHANGELOG.md
- Updated version number in README.md

### CODE REVIEW

File: sdk/communication/azure-communication-chat/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Release History
-## 1.0.0-beta.9 (Unreleased)
+## 1.0.0 (2021-06-09)
+### Dependency Updates
```
Comment (vcolin7) [inline sdk/communication/azure-communication-chat/CHANGELOG.md:3]: nit; I think we can move this section to the end of this section, users are probably more interested to know about other changes.

File: sdk/communication/azure-communication-chat/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+- Updated Azure Core from 1.0.0-beta.5 to 1.0.0-beta.6
```
Comment (vcolin7) [inline sdk/communication/azure-communication-chat/CHANGELOG.md:4]: Just a small suggestion:

```suggestion
- Updated `com.azure.android.core` from `1.0.0-beta.5` to `1.0.0-beta.6`.
```

File: sdk/communication/azure-communication-common/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Release History
 
-#1.1.0-beta1 (Unreleased)
+## 1.0.1 (2021-06-09)
+### Dependency Updates
+- Updated Azure Core from 1.0.0-beta.5 to 1.0.0-beta.6
```
Comment (vcolin7) [inline sdk/communication/azure-communication-common/CHANGELOG.md:5]: Just a small suggestion:

```suggestion
- Updated `com.azure.android.core` from `1.0.0-beta.5` to `1.0.0-beta.6`.
```

File: sdk/communication/azure-communication-chat/README.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ Azure Communication Chat contains the APIs used in chat applications for Azure C
 <dependency>
```
Comment (vcolin7) [inline sdk/communication/azure-communication-chat/README.md:20]: I think we need to change this **Include the package** section to look more like Common's **Install the library**, where they also include how to add the package to your project using gradle.

File: sdk/communication/azure-communication-chat/README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# # Azure Communication Chat client library for Java
+# Azure Communication Chat client library for Android
```
Comment (vcolin7) [inline sdk/communication/azure-communication-chat/README.md:1]: Good catch!

File: sdk/communication/azure-communication-chat/README.md [hunk 6]
```diff
@@ -13,14 +13,39 @@ Azure Communication Chat contains the APIs used in chat applications for Azure C
 - [Java Development Kit (JDK)](https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable) version 8 or above.
 - [Apache Maven](https://maven.apache.org/download.cgi).
 - A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
+
```
Comment (vcolin7) [inline sdk/communication/azure-communication-chat/README.md:16]:
```suggestion

### Versions available
The current version of this library is **1.0.0**.

```

**Review by vcolin7 — APPROVED** [review]

**Comment by JianpingChen** [comment]
Don't you need to change Chat build.gradle for version 1.0.0? Currently it says version = "1.0.0-beta.9"

**Review by vcolin7 — CHANGES_REQUESTED** [review]
Let's add the version changes to build.gradle files

**Review by JianpingChen — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -20,22 +20,24 @@ Currently, the client libraries are in **beta**. These libraries follow the [Azu
...
+- [azure-core-http](https://github.com/Azure/azure-sdk-for-android/tree/master/sdk/core/azure-core-http): 1.0.0-beta.6
+- [azure-core-http-okhttp](https://github.com/Azure/azure-sdk-for-android/tree/master/sdk/core/azure-core-http-okhttp): 1.0.0-beta.6
+- [azure-core-http-httpurlconnection](https://github.com/Azure/azure-sdk-for-android/tree/master/sdk/core/azure-core-http-httpurlconnection): 1.0.0-beta.6
+- [azure-core-jackson](https://github.com/Azure/azure-sdk-for-android/tree/master/sdk/core/azure-core-jackson): 1.0.0-beta.6
+- [azure-core-rest](https://github.com/Azure/azure-sdk-for-android/tree/master/sdk/core/azure-core-rest): 1.0.0-beta.6
+- [azure-core-credentials](https://github.com/Azure/azure-sdk-for-android/tree/master/sdk/core/azure-core-credential): 1.0.0-beta.6
 
 > Note: The SDK is currently in **beta**. The API surface and feature sets are subject to change at any time before they become generally available. We do not currently recommend them for production use.
 
+The following libraries are currently **general available**:
```
Comment (vcolin7) [inline README.md:34]:
```suggestion
The following libraries are currently **generally available**:
```

**Review by vcolin7 — APPROVED** [review]

**Pull request merged**